.buildkite/scripts/imageprep.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Do we expect to get more functionality into this script?

It might be better to incorporate these functions directly into `.buildkite/scripts/build_image.sh`.

Reply:
done